app/src/main/java/com/framgia/fsalon/FSalonApplication.java

```diff
@@ -10,6 +10,7 @@
  * Created by MyPC on 20/07/2017.
  */
 public class FSalonApplication extends Application {
+    private final static int a =10;
```

Comment:
ERROR: Static final field names must either be all caps (e.g. int HEIGHT_PX) for true constants, or start with s (e.g. AtomicInteger sNextId or Runnable sSuspendTask) for fields with mutable state or that dont feel like constants.